### ISSUE TITLE
add function country_name(country)

### ISSUE THREAD

**Issue by rapimo** [adjust/pg-country#8]
allow to display the full country name with something like

```
contrib_regression=# SELECT  country_name('de');
 country_name
--------------
 Germany
(1 row)
``` 

based on mapping [here](https://www.iban.com/country-codes)

**Comment by zilder** [comment]
Looks like it has been introduced in commit `d6431f539868e9c7335360c7a9da80acdf6d2ccb`. Should we close this?

**Comment by rapimo** [comment]
yep closed by #9
